buildbot.mariadb.org/master.cfg

```diff
@@ -3029,22 +3029,20 @@ f_dockerlibrary.addStep(steps.ShellCommand(
 f_dockerlibrary.addStep(steps.ShellCommand(
     name="building and test docker library image for MariaDB",
     command=["bash", "-xc", util.Interpolate("./docker-library-build-and-test.sh \"%(prop:tarbuildnum)s\" \"%(prop:mariadb_version)s\" \"%(prop:parentbuildername)s\"")]))
-
 ## f_aix
 f_aix = util.BuildFactory()
-f_aix.addStep(steps.SetProperty(property="dockerfile", value=util.Interpolate("%(kw:url)s", url=dockerfile), description="dockerfile"))
 f_aix.addStep(downloadSourceTarballAIX())
-f_aix.addStep(steps.ShellCommand(command=util.Interpolate("tar -xvzf /mnt/packages/%(prop:tarbuildnum)s_%(prop:mariadb_version)s.tar.gz --strip-components=1")))
+f_aix.addStep(steps.ShellCommand(command=util.Interpolate("tar -xvzf /mnt/packages/%(prop:tarbuildnum)s_%(prop:mariadb_version)s.tar.gz")))
 f_aix.addStep(steps.ShellCommand(name="create html log file", command=['bash', '-c', util.Interpolate(getHTMLLogString(), jobs=util.Property('jobs', default='6'))]))
+f_aix.addStep(steps.ShellCommand(name="fetch build script", command=['/opt/freeware/bin/curl', '--etag-compare', 'aix-script-tag', '--etag-save', 'aix-script-tag', 'https://raw.githubusercontent.com/MariaDB/mariadb.org-tools/master/buildbot.mariadb.org/scripts/aix-build-and-test.sh', '--output', 'aix-build-and-test.sh']))
 # build steps
 f_aix.addStep(steps.Compile(command=
-    ["sh", "-c", util.Interpolate("export TMPDIR=$HOME/tmp && export LIBPATH=/opt/freeware/lib/pthread/ppc64:/opt/freeware/lib:/usr/lib && cmake . -DCMAKE_BUILD_TYPE=%(kw:build_type)s -DCMAKE_C_COMPILER=%(kw:c_compiler)s -DCMAKE_CXX_COMPILER=%(kw:cxx_compiler)s -DCMAKE_AR=/usr/bin/ar -DLIBXML2_LIBRARY=/opt/freeware/lib/libxml2.a  -DPLUGIN_TOKUDB=NO -DPLUGIN_MROONGA=NO -DPLUGIN_SPIDER=NO -DPLUGIN_OQGRAPH=NO -DPLUGIN_PERFSCHEMA=%(kw:perf_schema)s -DPLUGIN_SPHINX=NO %(kw:additional_args)s -DWITH_UNIT_TESTS=NO -DPLUGIN_S3=NO -DWITH_MARIABACKUP=NO -DPLUGIN_WSREP_INFO=NO && make -j%(kw:jobs)s", perf_schema=util.Property('perf_schema', default='YES'), build_type=util.Property('build_type', default='RelWithDebInfo'), jobs=util.Property('jobs', default='3'), c_compiler=util.Property('c_compiler', default='gcc'), cxx_compiler=util.Property('cxx_compiler', default='g++'), additional_args=util.Property('additional_args', default='') )], env={'CCACHE_DIR':'/mnt/ccache'}, haltOnFailure="true"))
-
+    ["sh", "-c", util.Interpolate("chmod a+x ./aix-build-and-test.sh && ./aix-build-and-test.sh build %(prop:mariadb_version)s %(kw:build_type)s", build_type=util.Property('build_type', default='RelWithDebInfo'))], haltOnFailure="true"))
```

Comment:
I guess the "jobs" property should be passed as an argument to the build script

Reply:
probably jobs should remain in the script so it can be adjusted without a master reload.

Reply:
However, if the value is different on the master side, it will not save all the MTR logs/search for files that do not exist. This would only happen for AIX though. I think that it would be slightly better to have only one value, but totally agree that it would be nice not to require a master restart. Maybe we can change the script to get the master value once we set a proper one. Anyway, if you think this is not an issue, feel free to merge the PR.

Reply:
b90fa95a7ba317b012e8c7077fad2235d08fe560 - 12 seems to be the number of mtr jobs currently. I tested the build and it can handle 24.